spec/helpers/posts_helper_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+# Specs in this file have access to a helper object that includes
+# the PostsHelper. For example:
+#
+# describe PostsHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
+RSpec.describe PostsHelper, type: :helper do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Setup_Controllers_project_v1_1.md#:~:text=Important%3A-,do%20NOT%20use%20rails%20generate%20scaffold%20command.,-The%20point%20of), You specifically asked not to use `rails generate scaffold` command and you used it anyway.
Please remove excess files created by the command like:
   - All created files in `./app/helpers`.
   - All created files in `./spec`.
   
   
![image](https://github.com/ReemMohamedAbdelfatah/rails-blog-app/assets/68796232/873775c1-d4a8-4e94-852c-b5242cb15e56)

![image](https://github.com/ReemMohamedAbdelfatah/rails-blog-app/assets/68796232/11c43a94-a85d-4826-a539-eeb52db1a311)

Reply:
I used "rails generate controller Users index show" is it considered scaffold?